cernopendata_client/searcher.py

```diff
@@ -43,7 +43,7 @@ def verify_recid(server=None, recid=None):
     else:
         try:
             input_record_url_check.raise_for_status()
-        except requests.HTTPError:
+        except Exception:
             display_message(
                 msg_type="error",
                 msg="The record id number you supplied is not valid.",
```

Comment:
'record id' -> 'record ID'

Can you please update this part, even if it is not part of the PR?  Also in other places:

```console
$ rg 'record id'
cernopendata_client/searcher.py
34:    :return: Boolean after verifying the record id
49:                msg="The record id number you supplied is not valid.",
73:            msg="The record id number you supplied is not valid.",
80:    """Return record id by either title or doi.
89:    :return: record id

cernopendata_client/validator.py
29:            msg="You must supply a record id number as an " "input using -r flag.",
```

Could be a separate commit.